extensions/sql-migration/src/dialog/help/helpAndSupportDialog.ts

```diff
@@ -0,0 +1,171 @@
+			description: constants.SUPPORT_RESOURCES_DESCRIPTION,
+			link: '',
+			iconPath: IconPathHelper.info,
+		},
+		{
+			title: constants.SUPPORT_REQUEST_TITLE,
+			description: constants.SUPPORT_REQUEST_DESCRIPTION,
+			link: '',
+			iconPath: IconPathHelper.newSupportRequest,
+		},
+		{
+			title: constants.COMMUNITY_SUPPORT_TITLE,
+			description: constants.COMMUNITY_SUPPORT_DESCRIPTION,
+			link: '',
+			iconPath: IconPathHelper.newSupportRequest,
+		}];
+
+		linksContainer.addItems(support.map(l => this.createSupportContainer(view, l)));
+
+		return linksContainer;
+	}
+
+	private createSupportContainer(view: azdata.ModelView, linkMetaData: IActionMetadata): azdata.Component {
+		const maxWidth = 474;
+		const labelsContainer = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'flex-direction': 'column',
+				'width': `${maxWidth}px`,
+				'justify-content': 'flex-start',
+				'margin-bottom': '12px'
+			}
+		}).component();
+
+		const titleComponent = view.modelBuilder.text().withProps({
+			value: linkMetaData.title,
+			CSSStyles: {
+				'font-size': '14px',
+				'line-height': '20px',
+				'font-weight': '600',
+				'margin-bottom': '5px',
+			}
+		}).component();
+
+		labelsContainer.addItems([titleComponent]);
+
+		const linksContainer = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'flex-direction': 'row',
+				'width': `${maxWidth + 10}px`,
+				'justify-content': 'flex-start',
+			}
+		}).component();
+
+		const linkImageComponent = view.modelBuilder.image().withProps({
+			iconPath: linkMetaData.iconPath,
+			iconHeight: '16px',
+			iconWidth: '16px',
+			width: '16px',
+			height: '16px',
+
+		}).component();
+		linksContainer.addItems([linkImageComponent]);
+
+		const linkComponent = view.modelBuilder.hyperlink().withProps({
+			label: linkMetaData.description,
+			url: linkMetaData.link,
+			showLinkIcon: true,
+			CSSStyles: {
+				'font-size': '13px',
+				'line-height': '18px',
+				'margin-left': '-225px',
+				'text-decoration': 'none'
+			}
+		}).component();
+
+		const note = view.modelBuilder.text().withProps({
+			value: constants.SUPPORT_REQUEST_NOTE,
+			CSSStyles: {
+				'margin-block-end': '-1px'
+			},
+		}).component();
+
+		const communitySupportNote = view.modelBuilder.text().withProps({
+			value: constants.COMMUNITY_SUPPORT_NOTE,
+			CSSStyles: {
+				'margin-block-end': '-1px'
+			},
+		}).component();
+
+		linksContainer.addItems([linkComponent]);
+		labelsContainer.addItems([linksContainer]);
+
+		if (linkMetaData.title === constants.SUPPORT_REQUEST_TITLE) {
+			linkComponent.onDidClick(async () => await this.launchNewSupportRequest());
+			labelsContainer.addItem(note);
+		}
+		else if (linkMetaData.title === constants.COMMUNITY_SUPPORT_TITLE) {
+			linkComponent.onDidClick(async () => await this.launchCommunityRequest());
+			labelsContainer.addItem(communitySupportNote);
+		}
+		else
+			linkComponent.url = 'https://aka.ms/dms-overview'
```

Comment:
Should we define all the links as constants at a single place and use them?

Reply:
Done